content/sensu-go/5.0/reference/rbac.md

```diff
@@ -1,85 +1,407 @@
-default      | current environment value configured for `sensuctl` (ie `default`)
-example      | {{< highlight shell >}}"organization": "default"{{</highlight>}}
+description  | The rulesets that a role applies.
+required     | true 
+type         | Array 
+example      | {{< highlight shell >}}"rules": [
+  {
+    "verbs": ["get", "list"],
+    "resources": ["checks"],
+    "resourceNames": [""]
+  }
+]{{< /highlight >}}
+
+#### Rule attributes
+A rule is an explicit statement which grants a particular permission to a resource.
 
-### User
+verbs  | 
+-------------|------ 
+description  | The permissions to be applied by the rule: `create`, `get`, `update`, or `delete`. 
+required     | true 
+type         | Array
+example      | {{< highlight shell >}}"verbs": ["get", "list"]{{< /highlight >}}
 
+resources         | 
+-------------|------ 
+description  | The type of resource that the rule has permission to access. Roles can only access [namespaced resource types][17] while cluster roles can access namespaced and [cluster-wide resource types][18]. See [resource types][4] for available types.
+required     | true 
+type         | Array
+example      | {{< highlight shell >}}"resources": ["checks"]{{< /highlight >}}
+
+resourceNames    | 
+-------------|------ 
+description  | Specific resource names that the rule has permission to access. Resource name permissions are only available for `get`, `delete`, and `update` verbs.
+required     | false
+type         | Array
+example      | {{< highlight shell >}}"resourceNames": ["check-cpu"]{{< /highlight >}}
+
+## Users
 A user represents a person or an agent which interacts with Sensu.
+You can use your Sensu username and password to [configure sensuctl][26] or log in to the [dashboard][3].
+
+### Default user
+
+By default, Sensu includes a global `admin` user that you can use to manage Sensu and create new users.
+
+| attribute | value |
+| --------- | ----- |
+| username   | `admin`  |
+| password   | `P@ssw0rd!`  |
+| groups   | `cluster-admins`  |
+| cluster role   |  `cluster-admin` |
+| cluster role binding   | `cluster-admin	`  |
+
+We **strongly** recommended changing the default password for the admin user immediately.
+Once authenticated, you can change the password using the `change-password` command.
+
+{{< highlight shell >}}
+sensuctl user change-password
+{{< /highlight >}}
+
+Sensu also includes an `agent` user that is used internally by the Sensu agent and should not be modified.
+
+_WARNING: Modification of the `agent` user can result in non-functional Sensu agents._
+
+### Viewing users
+You can use [sensuctl][2] to see a list of all users within Sensu:
+
+{{< highlight shell >}}
+sensuctl user list
+{{< /highlight >}}
+
+### Creating a user
+You can use [sensuctl][2] to create a user.
+For example, the following command creates a user with the username `alice` and the password `P@ssw0rd!`.
+
+{{< highlight shell >}}
+sensuctl user create alice --password P@ssw0rd!
```

Comment:
Probably safer to add quotes around the password because it contains special characters:

`sensuctl user create alice --password 'P@ssw0rd!'`